central/imageintegration/datastore/telemetry.go

```diff
@@ -0,0 +1,67 @@
+package datastore
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/registries/types"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	"github.com/stackrox/rox/pkg/telemetry/phonehome"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+// Gather image integration types.
+// Current properties we gather:
+// "Total Image Integrations"
+// "Total <integration type> Image Integrations"
+var Gather phonehome.GatherFunc = func(ctx context.Context) (map[string]any, error) {
```

Comment:
Is this used yet?

Reply:
Ah good point I missed a call to add the gatherer.